buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -286,8 +286,12 @@ def build_with_slack(DOWNSTREAM_JOB_NAME, ghprbGhRepository, ghprbActualCommit,
     // Set Github Commit Status
     if (ghprbActualCommit) {
         node(SETUP_LABEL) {
-            set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, BUILD_URL, 'PENDING', "Build Started")
-        }
+            try {
+                retry_and_delay({set_build_status(GITHUB_REPO, DOWNSTREAM_JOB_NAME, ghprbActualCommit, BUILD_URL, 'PENDING', "Build Started")})
+            } catch (e) {
+                println "set_build_status failed"
+            }   
```

Comment:
Some of these lines have extra whitespace at the end of the line. Please cleanup then this looks good to me.

Reply:
Hey @AdamBrousseau! Thanks for checking! I believe it is clean now. Thanks!